using-pagination/src/pages/index.js

```diff
@@ -1,13 +1,40 @@
+import Link from 'next/link';
 import { gql } from 'graphql-request';
 
 import { graphcms } from '../../lib/_graphcms';
 
 const limit = 1;
 
+const singleProductStyle = {
```

Comment:
We want to improve styling for all our examples. I'm happy if you want to add these here, but FYI, these may get removed with an updated design when we get some design resources to help here.

Reply:
That's definitely a thing! Do you already have basic styling from other example that I could use here?

Reply:
> these may get removed with an updated design when we get some design resources to help here